app/views/posts/index.html.erb

```diff
@@ -1 +1,35 @@
-<h1>Posts</h1>
\ No newline at end of file
+<div>
+    <div class='btn-container'><%= link_to "Back to User", @user, class:'back' %></div>
+    <div class="user">
+        <div class="user-image">
+            <%= image_tag @user.photo %>
+        </div>
+        <div class="user-info">
+            <h1><%= @user.name %></h1>
+            <h3>Number of posts: <%= @user.posts_counter %></h3>
+        </div>
+    </div>
+    <div class="posts">
+        <% @posts.each do |post| %>
+        <%= link_to "/users/#{@user.id}/posts/#{post.id}" do %>
+        <div class="post">
+            <h2><%= post.title %></h2>
+            <p><%= post.text %></p>
+            <div>
+                <p>Comments: <%= post.comments_counter %></p>
+                <p>Likes: <%= post.likes_counter %></p>
+            </div>
+        </div>
+        <% if post.comments.length > 0 %>
+        <div class="comments">
+            <% post.recent_comments.each do |comment| %>
+                <div class="comment">
+                    <p><%= comment.user.name %>: <%= comment.text %></p>
+                </div>
+            <% end %>
+        </div>
+        <% end %>
+        <% end %>
+        <% end %>
+    </div>
+</div>
```

Comment:
![image](https://github.com/joramot/BlogApp/assets/93039359/8c06a721-c790-41cc-8e29-77f6398a0d1a)

- [ ] Great work so far but it would be nice if you added the Pagination button at the end of the page to make your project  look similar to the given design :+1: .